extensions/pkg/controller/worker/genericactuator/actuator_reconcile.go

```diff
@@ -418,14 +415,6 @@ func (a *genericActuator) waitUntilWantedMachineDeploymentsAvailable(ctx context
 				return retryutils.Ok()
 			}
 
-			// update worker status with condition that indicates an ongoing rolling update operation
-			if !workerStatusUpdatedForRollingUpdate {
-				if err := a.updateWorkerStatusMachineDeployments(ctx, worker, extensionsworker.MachineDeployments{}, true); err != nil {
```

Comment:
Why the whole `updateWorkerStatusMachineDeployments` call is removed? Don't we still need to update the `status.machineDeployments` field?